cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
+import { generateUser, generateeArticle } from '../support/generate';
 
+describe('Article flow', () => {
+  const user = generateUser();
+  const article = generateeArticle();
```

Comment:
The function `generateeArticle` is misspelled. It should be `generateArticle` to match the import statement.